src/styles/page.css

```diff
@@ -0,0 +1,3 @@
+.page {
```

Comment:
```suggestion
.html {
```

rename on 'normalize'